packages/dita-example-sx-modules-spec-learning-xsd-learning2domain/src/api/insertNodeAndRemoveFromSiblingsCustomMutation.js

```diff
@@ -28,11 +29,11 @@ export default function insertNodeAndRemoveFromSiblings(argument, blueprint) {
 
 	blueprint.insertBefore(contextNode, newNode, referenceNode);
 
-	evaluateXPathToNodes('preceding-sibling::*', contextNode, blueprint)
-		.concat(evaluateXPathToNodes('following-sibling::*', contextNode, blueprint))
+	evaluateXPathToNodes(xq`preceding-sibling::*`, contextNode, blueprint)
+		.concat(evaluateXPathToNodes(xq`following-sibling::*`, contextNode, blueprint))
 		.forEach(function(siblingNode) {
 			var removeNode = evaluateXPathToFirstNode(
-				'child::' + argument.nodeName,
+				xq(`child::${argument.nodeName}`),
```

Comment:
Lets add a comment here:

```
Use the xq() function to generate an XPath test from this expression
```

or even better: change the query to `child::*[name() = ${argument.nodeName}]` instead! Preventing string interpolation is always good!